handlers/main.yml

```diff
@@ -1,10 +1,7 @@
-# SPDX-License-Identifier: MIT
 ---
-# Put handlers for tasks here.
-
-# Example:
-- name: template handler to restart services
+- name: enable and restart vpn services
   service:
-    name: "{{ item }}"
-    state: restarted
-  loop: "{{ __template_services }}"
+    name: ipsec
```

Comment:
you should use `__vpn_services` with the loop as in the template, where `__vpn_services` is set in `vars/main.yml` - also in `cleanup` below